applications/GeoMechanicsApplication/tests/cpp_tests/test_settlement_workflow.cpp

```diff
@@ -26,15 +26,18 @@ using namespace Kratos;
 namespace Kratos::Testing
 {
 
-[[maybe_unused]] void TestSettlement()
+KRATOS_TEST_CASE_IN_SUITE(SettlementWorkflow, KratosGeoMechanicsFastSuite)
```

Comment:
Don't forget to move this back to [[maybe_unused]] void TestSettlement() before merging (okay it will block, so probably you can't forget)

Reply:
Thanks for the reminder. I have rolled back the changes such that the build checks will pass again.